gulpfile.js

```diff
@@ -23,13 +23,13 @@ var paths = {
 gulp.task("build-js", () => {
     return gulp
         .src(paths.scripts)
+        .pipe(replace("__VERSION__", _package.version))

```

Comment:
isn't this going to change the source (non packed) files also ?

Reply:
The changes occur on files on the `dist` folder